pysixtrack/track.py

```diff
@@ -318,52 +316,75 @@ def tobuffer(self):
 
 class BeamBeam6D(Element):
     __slots__ = ([
-        'q_part', 'phi', 'alpha', 'delta_x', 'delta_y',
-        'N_part_per_slice', 'z_slices',
-        'Sig_11_0', 'Sig_12_0', 'Sig_13_0',
-        'Sig_14_0', 'Sig_22_0', 'Sig_23_0',
-        'Sig_24_0', 'Sig_33_0', 'Sig_34_0', 'Sig_44_0',
-        'x_CO', 'px_CO', 'y_CO', 'py_CO', 'sigma_CO', 'delta_CO',
-        'min_sigma_diff', 'threshold_singular',
-        'Dx_sub', 'Dpx_sub', 'Dy_sub', 'Dpy_sub', 'Dsigma_sub', 'Ddelta_sub',
+        'phi',
+        'alpha',
+        'x_bb_co',
+        'y_bb_co',
+        'charge_slices',
+        'zeta_slices',
+        'sigma_11',
+        'sigma_12',
+        'sigma_13',
+        'sigma_14',
+        'sigma_22',
+        'sigma_23',
+        'sigma_24',
+        'sigma_33',
+        'sigma_34',
+        'sigma_44',
+        'x_co',
+        'px_co',
+        'y_co',
+        'py_co',
+        'zeta_co',
+        'delta_co',
+        'd_x',
+        'd_px',
+        'd_y',
+        'd_py',
+        'd_zeta',
+        'd_delta',
+        'min_sigma_diff',
+        'threshold_singular',
         'enabled'])
+
     __units__ = tuple(len(__slots__) * [[]])
-    __defaults__ = tuple(len(__slots__) * [0.])
+    __defaults__ = tuple((len(__slots__)-3) * [0.] + [1e-28, 1e-28, True])
 
     def track(self, p):
         if self.enabled:
             bb6data = BB6Ddata.BB6D_init(
-                self.q_part,
+                qe,
```

Comment:
dov'è definito `qe`?

Reply:
It's imported from scipy.constants at the beginning of the file